context/app/routes_api.py

```diff
@@ -1,23 +1,45 @@
 from io import StringIO
 from csv import DictWriter
 
-from flask import Response, abort, request, render_template
+from flask import Response, abort, request, render_template, jsonify
 
 from .utils import make_blueprint, get_client, get_default_flask_data
 
 
 blueprint = make_blueprint(__name__)
 
 
-@blueprint.route('/metadata/v0/<entity_type>.tsv')
+def _get_dict_subset(d, keys_to_remove):
```

Comment:
Consider renaming: Right now, just from the function name, it's not clear whether the given keys are the ones that stay or go. I might do:
```suggestion
def _drop_keys(d, keys_to_remove):
```